src/ribbon/buttonbar.cpp

```diff
@@ -1532,6 +1560,15 @@ wxRect wxRibbonButtonBar::GetItemRect(int button_id)const
     return wxRect();
 }
 
+void wxRibbonButtonBar::SetButtonBitmapSize(const wxSize& large_btn, const wxSize& small_btn)
+{
+    if (m_buttons.IsEmpty())
```

Comment:
This should be an assert, i.e.
```suggestion
    wxCHECK_RET( m_buttons.IsEmpty(), "Must be called before adding any buttons" );
```
as silently ignoring the provided values is really not very developer-friendly.